Dockerfile

```diff
@@ -71,6 +71,9 @@ RUN --mount=type=bind,target=. \
   xx-go install std
   xx-go build -ldflags "$(cat /tmp/.ldflags)" -o /out/docker-credential-osxkeychain-${TARGETARCH}${TARGETVARIANT} ./osxkeychain/cmd/main_darwin.go
   xx-verify /out/docker-credential-osxkeychain-${TARGETARCH}${TARGETVARIANT}
+
+  xx-go build -ldflags "$(cat /tmp/.ldflags)" -o /out/docker-credential-pass-${TARGETOS}-${TARGETARCH}${TARGETVARIANT} ./pass/cmd/main.go
+  xx-verify /out/docker-credential-pass-${TARGETOS}-${TARGETARCH}${TARGETVARIANT}
```

Comment:
I think you need to define `ARG TARGETOS` in this stage too.

Reply:
🤦 that's what you get when you try to do it too fast 😂